pkg/core/app/app.go

```diff
@@ -65,11 +66,32 @@ type Options struct {
 	DockerNetwork string
 }
 
+func findComposeFilePathFromCmdArgs(args []string) string {
```

Comment:
`findComposeFile` should handle all the logic of this, no need to make a separate function.

Reply:
Please remove the older function